test/Grpc.Net.Client.Tests/Grpc.Net.Client.Tests.csproj

```diff
@@ -31,10 +32,4 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Testing" Version="$(MicrosoftExtensionsPackageVersion)" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Not sure why I added this override but this was the root cause of the linked issue.